### DISCUSSION TITLE
Where should categories go in the manga details page?

### DISCUSSION THREAD

**Discussion by NGB-Was-Taken** [Animeboynz/Mihon-Backup-Viewer#37] [Ideas]
I am thinking of making a button that opens a context menu listing the categories the manga is in. From there you can select a category to switch to that tab. 

If you want to do something else with the categories do let me know, else, any ideas on where the button should be?

@BrutuZ @Animeboynz

**Comment by Animeboynz** [comment]
I'm not too sure what you mean but, Once we can see some visual ideas for this we can probably help.

**Comment by BrutuZ** [comment]
I don't think a button is a good way, seems like an unnecessary interaction.
If you really want the switch view to category mechanic, why not make the indicator themselves the button? Give them the Tags/Genres style and on click it closes the details modal and switch the view

**Comment by Animeboynz** [comment]
Implemented in #39
